js/modules/modules.go

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2020 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package modules
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/loadimpact/k6/js/internal/modules"
+)
+
+const extPrefix string = "k6/x/"
+
+// Get returns the module registered with name.
+func Get(name string) interface{} {
+	return modules.Get(name)
+}
+
+// Register the given mod as an external JavaScript module,
+// available for import from JS scripts with the "k6/x/<name>" import path.
+// This function panics if a module with the same name is already registered.
+func Register(name string, mod interface{}) {
+	if !strings.HasPrefix(name, extPrefix) {
+		name = fmt.Sprintf("%s%s", extPrefix, name)
+	}
```

Comment:
I kind of wonder if we shouldn't just either:
1. always prefix it 
2. panic if it isn't prefixed instead of prefixing it.

My reasoning is that if someone writes `Register("k7/x/something")` and then goes to use their module they will be really surprised that it totally doesn't work. While if it panics or the documentation specifically says taht we always prepend `k6/x/` that will be much better IMO

Reply:
Hhmm well the function comment does explain that imports will be in the form `k6/x/<name>`, and this gives the option to register with or without it. I can see how panicking would be helpful though, but not sure if we should force users to specify the prefix. Kind of torn on this one...

Reply:
I think it would be better to just ask for the name and always prepend the prefix ourselves. We can simply mention this approach in the docs and our reasoning would be that it is the intended behavior. Then there's no need for panic.

Reply:
Automatically adding prefixes will likely just lead to more confusion.

I'm thinking that manual import path config + Panics would be preferable as it:

1. The error tells the user how to solve it.
2. No hidden behavior you need to know or document. 
3. Import path in the JS file will match exactly what they have written in their plugin.

Reply:
OK, @simskij convinced me, `panic` it is. 😄

Will push the change in a bit.

Reply:
as before I don't think categories are needed for k6. I find the "we have different module registers and functions for different categories" approach much better and easier to document and easy for the people who want add 1 type of plugin to not have to read how it's done for the others.

We can always add a more central plugin system and wire the old calls to it and recommend that people use the new system ... although again ... I highly doubt this day will come